### PR TITLE
[3.0][Cookbook] Use the 3.0 directory structure

### PR DESCRIPTION
**Pull request by wouterj**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | 3.0+ |
| Fixed tickets | part of https://github.com/symfony/symfony-docs/issues/5898 |

### CODE REVIEW

**Comment by javiereguiluz** [comment]
:+1:

File: cookbook/configuration/configuration_organization.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ default Symfony Standard Edition follow this structure:
 
 .. code-block:: text
 
-    <your-project>/
+    your-project/
```
Comment (ogizanagi) [inline cookbook/configuration/configuration_organization.rst]: Why isn't the `bin` directory present in here ? Is it expected ?
It has nothing to do with configuration files, but same as for new `var` and `tests` directories, right ? (Well, eventually `var` for environments)

File: cookbook/configuration/override_dir_structure.rst [hunk 2]
```diff
@@ -12,12 +12,16 @@ directory structure is:
 
     your-project/
```
Comment (ogizanagi) [inline cookbook/configuration/override_dir_structure.rst]: Same here (https://github.com/symfony/symfony-docs/pull/5917/files#r46079205)

File: cookbook/deployment/azure-website.rst [hunk 3]
```diff
@@ -260,12 +260,12 @@ directory with at least the following contents:
...
+    /var/cache/*
     /app/config/parameters.yml
-    /app/logs/*
-    !app/cache/.gitkeep
-    !app/logs/.gitkeep
-    /app/SymfonyRequirements.php
+    /var/logs/*
+    !var/cache/.gitkeep
+    !var/logs/.gitkeep
+    /bin/SymfonyRequirements.php
```
Comment (ogizanagi) [inline cookbook/deployment/azure-website.rst]:
```diff
- /bin/SymfonyRequirements.php
+ /var/SymfonyRequirements.php
```

?

**Comment by WouterJ** [comment]
Thanks @ogizanagi for the review, I applied your suggestions.

File: cookbook/configuration/configuration_organization.rst (hunk 1 above)
Comment (weaverryan) [inline cookbook/configuration/configuration_organization.rst]: I read through it: it's focusing on *just* the stuff in the `app/` directory. I think really we never needed `src`, `vendor` or `web` here.

File: cookbook/configuration/override_dir_structure.rst (hunk 2 above)
Comment (weaverryan) [inline cookbook/configuration/override_dir_structure.rst]: I added `bin/` to this one

**Comment by weaverryan** [comment]
@WouterJ YOU ROCK! I did not have time to do this before this week, so I was thrilled to see these! It's impossible to catch everything with these big changes, so I did my own searched after merging and found a few more things, but not too much: sha: 3083e90a560af5dd3134b568a4d8a5de9babd183. That gives me pretty good confidence :)

**Comment by taylankasap** [comment]
Why/how did this affect 2.7 & 2.8? [Example here](https://github.com/symfony/symfony-docs/blame/2.7/cookbook/assetic/asset_management.rst#L601)

**Comment by WouterJ** [comment]
@taylankasap a merge error I made yesterday. I'm on it, but it's quite a hard one to fix.

**Pull request merged**